testsuite/openshift/objects/auth_config/auth_policy.py

```diff
@@ -41,3 +41,10 @@ def add_rule(self, when: list[Rule]):
         """Add rule for the skip of entire AuthPolicy"""
         self.model.spec.setdefault("when", [])
         self.model.spec["when"].extend([asdict(x) for x in when])
+
+    @modify
+    def add_patterns(self, patterns: dict[str, list[Pattern]]):
```

Comment:
Do you need to have it in AuthPolicy if it is already in AuthConfig?

Reply:
True, I will remove it from AuthPolicy